xprof/xprof.sh.erb.in

```diff
@@ -448,11 +448,11 @@ setub_babeltrace_live() {
 <% if languages.include?("omp") %>
      --component=filter.ompinterval.interval \
 <% end %>
-     --component=sink.tally.tally \
-     --params="display=compact, \
-               name=demangle, \
-               display_mode=human, \
```

Comment:
Why did we switch to a bool for this one?
I think it make sense to have `text/human` or `json` ?